paasta_tools/cli/cmds/status.py

```diff
@@ -189,6 +189,11 @@ def add_instance_filter_arguments(status_parser, verb: str = "inspect") -> None:
     status_parser.add_argument(
         "-r", "--registration", help=f"Only {verb} instances with this registration."
     )
+    status_parser.add_argument(
+        "-t",
+        "--instance-type",
+        help=f"Override the inferred instance type. By default, instance type is inferred from status object.",
```

Comment:
Could you please add a couple of examples of instance-type just so it's super easy for paasta users to understand what we mean by instance-type.

Reply:
might even be able to use https://docs.python.org/3/library/argparse.html#choices (and point it at https://sourcegraph.yelpcorp.com/Yelp/paasta/-/blob/paasta_tools/instance/kubernetes.py?subtree=true#L51) so that users can't pass in invalid values

Reply:
@mattmb @nemacysts 
Thank you for your suggestions! I simplified the implementation by replacing
```
--instance-type=kubernetes
```
with a simple boolean flag `--kubernetes` / `-k` for printing the status using the kubernetes writer when it is available. Overriding instance type was too confusing and was covering cases where it made no sense, but I hope the new flag self-descriptive :) Also, listed the CRs where this flag would be useful in the help message.

Reply:
The flag was removed entirely. We will print all statuses that are available.